### DISCUSSION TITLE
Where does Podman Desktop for macOS store its volumes

### DISCUSSION THREAD

**Discussion by brendon** [podman-desktop/podman-desktop#1659] [Q&A] [answered]
I'm having a hard time figuring out where volumes of type `persistentVolumeClaim` are stored on the macOS operating system. All the info I can find tells me where they are in the VM but I'd like to exclude these volumes from Time Machine backups as one of my volumes is quite large. Does anyone know? :)

**Comment by mairin** [comment, marked as answer]
Hi @brendon! Sorry for the slow reply - on macOS, the VMs should be in your home directory. Podman Desktop installs them to ~/.local/share/containers/podman/machine. Let us know if this helps or if you need more help!

**Comment by brendon** [comment]
Thanks @mairin, that's super helpful. Thank you very much! :D
